src/nunit.analyzers/DisposeFieldsAndPropertiesInTearDown/DisposeFieldsAndPropertiesInTearDownAnalyzer.cs

```diff
@@ -149,6 +149,22 @@ private static void AnalyzeDisposableFields(SyntaxNodeAnalysisContext context)
                 disposeMethods = disposeMethods.Union(value.Split(AdditionalDisposalMethodsSeparators, StringSplitOptions.RemoveEmptyEntries));
             }
 
+            // Sometimes virtual method are called from base class SetUp/TearDown methods.
```

Comment:
Can the method be `abstract` as well?

Reply:
No. The base class methods can be `abstract`, but the analyzer checks the derived class and there the methods must be `override` as they need to be callable from the base class.